src/components/features/forms/newDeliveryForm/NewDeliveryStep5Form.tsx

```diff
@@ -0,0 +1,118 @@
+            <Typography>{formsData.deliveryNumber}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step1.receptionNumber')}</Typography>
+            <Typography>{formsData.receptionNumber}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step1.cmrNumber')}</Typography>
+            <Typography>{formsData.cmrNumber}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step1.markers')}</Typography>
+            <ChipsList items={formsData.markers} />
+          </Box>
+        </Box>
+      </Box>
+
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1em' }}>
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: '1em' }}>
+          <ChipsList items={['2']} />
+          <Typography variant="h6">{translate('newDelivery.steps.truckDetails')}</Typography>
+        </Box>
+
+        <Box sx={{ display: 'flex', gap: '2em' }}>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step2.vendorName')}</Typography>
+            <Typography>{formsData.vendorName}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step2.vendorId')}</Typography>
+            <Typography>{formsData.vendorId}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step2.truckNumber')}</Typography>
+            <Typography>{formsData.truckNumber}</Typography>
+          </Box>
+          <Box>
+            <Typography>{translate('newDelivery.labels.step2.deliveryDate')}</Typography>
+            <Typography>{dayjs(formsData.deliveryDate).format('DD.MM.YYYY')}</Typography>
+          </Box>
+        </Box>
+      </Box>
+
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1em' }}>
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: '1em' }}>
+          <ChipsList items={['3']} />
+          <Typography variant="h6">{translate('newDelivery.steps.goodDetails')}</Typography>
+        </Box>
+
+        <Box sx={{ display: 'flex', gap: '2em' }}>
+          {formsData.goods.map((good: any, index: number) => {
+            return (
+              <Box key={index}>
+                <Typography>{translate(`newDelivery.goodType.${good.goodTypeStep3}`)}</Typography>
+                <Typography>{good.goodQuantityStep3}</Typography>
+              </Box>
+            )
+          })}
+        </Box>
+      </Box>
+
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1em' }}>
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: '1em' }}>
+          <ChipsList items={['4']} />
+          <Typography variant="h6">{translate('newDelivery.steps.goodRelocate')}</Typography>
+        </Box>
+
+        <Box>
+          <NewDeliveryStep5Table />
+        </Box>
```

Comment:
we dont need step 5, only through step 4. You were looking at the design about `DeliveryDetails` which is another task, screen and place to show it

Reply:
Fixed